modules/sdk/project-templates/project-templates-portlet-provider/src/main/resources/archetype-resources/pom.xml

```diff
@@ -64,22 +64,6 @@
 	</dependencies>
 	<build>
 		<plugins>
-			<plugin>
```

Comment:
Was there a reason why you sorted this?  I don't know that it matters very much just wondering.